### ISSUE TITLE
Ability to display filter counters in front + Ability to remove parentheses in counters / jet smart filters

### ISSUE THREAD

**Issue by Michaelgimii** [CrocoBlock/suggestions#5571]
<strike>1- Please add a new feature so we can move the counters to the opposite side : https://prnt.sc/uGDX-_L4VczE</strike> DONE
<strike>2- Please add the option to delete parentheses to the counters. Counters are now displayed this way : (12)</strike> DONE
3- Please add 3 new options to the counters. Background color - padding - border radius . By adding these items, we can create something like this : https://prnt.sc/WtbJwxAsvWTw

Please add these items. Honestly, the ui in the filters of this plugin is extremely weak. So adding such items can greatly improve the ui.

**Comment by Lonsdale201** [comment]
Idea is good, btw all of this easy to achive a bit of custom css.

**Comment by Michaelgimii** [comment]
Yes, but many users of this plugin do not have coding knowledge. On the other hand, I do not find it interesting that crocoblock, for jet smart filters, displays features on its website that are not present in this plugin at all. Like this : https://prnt.sc/ML8DdfNtrxT_

**Comment by Lonsdale201** [comment]
> Yes, but many users of this plugin do not have coding knowledge. On the other hand, I do not find it interesting that crocoblock, for jet smart filters, displays features on its website that are not present in this plugin at all. Like this : https://prnt.sc/ML8DdfNtrxT_

Absolute true. In this image, you want to make a select filter more customizable?

**Comment by Michaelgimii** [comment]
> > Yes, but many users of this plugin do not have coding knowledge. On the other hand, I do not find it interesting that crocoblock, for jet smart filters, displays features on its website that are not present in this plugin at all. Like this : https://prnt.sc/ML8DdfNtrxT_
> 
> Absolute true. In this image, you want to make a select filter more customizable?

Yes, exactly

**Comment by Lonsdale201** [comment]
> > > Yes, but many users of this plugin do not have coding knowledge. On the other hand, I do not find it interesting that crocoblock, for jet smart filters, displays features on its website that are not present in this plugin at all. Like this : https://prnt.sc/ML8DdfNtrxT_
> > 
> > 
> > Absolute true. In this image, you want to make a select filter more customizable?
> 
> Yes, exactly

Ahh understand. You can do this with a little jquery extension calling select2. If you need i can can write a short desc how can you easy implement. :)
